policyengine_canada/parameters/gov/provinces/ab/tax/income/credits/disability/base.yaml

```diff
@@ -0,0 +1,12 @@
+description: Alberta provides this amount under the disability credit. 
+values:
+  2022-01-01: 16_201
+metadata:
+  unit: currency-CAD
+  period: year
+  label: Alberta disability credit amount
+  reference:
+  - title: Alberta disability credit base amount - Section 4
+    href: https://www.canada.ca/content/dam/cra-arc/formspubs/pbg/td1ab/td1ab-23e.pdf#page=1
```

Comment:
add this reference: https://www.canada.ca/content/dam/cra-arc/formspubs/pbg/5009-d/5009-d-22e.pdf